default/generated/spring-framework/spring-framework-5.x-to-6.0-data-access.yaml

```diff
@@ -90,3 +90,39 @@
       url: https://eclipse.dev/eclipselink/downloads/previous_releases.php
     - title: 'Spring 6.0 migration guide'
       url: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-6.0-Release-Notes#removed-apis
+
+- ruleID: spring-framework-5.x-to-6.0-data-access-00030
+  category: mandatory
+  effort: 1
+  labels:
+  - konveyor.io/source=springboot2
+  - konveyor.io/target=springboot3
```

Comment:
The targets would be:
```yaml
  labels:
    - konveyor.io/source=spring5
    - konveyor.io/target=spring6+
```

We'll have to add the spring boot targets eventually, but for now lets just use these.